xsuite-lightsimulnet/src/handleAdmin.go

```diff
@@ -148,6 +166,61 @@ func (e *Executor) setAccount(rawAccount RawAccount) error {
 	return nil
 }
 
+func (e *Executor) updateAccount(rawAccount UpdateRawAccount) error {
+	address, err := bech32Decode(rawAccount.Address)
+	if err != nil {
+		return err
+	}
+	worldAccount := e.getWorldAccount(address)
+	if rawAccount.Nonce != nil {
+		worldAccount.Nonce = *rawAccount.Nonce
+	}
+	if rawAccount.Balance != nil {
+		worldAccount.Balance, err = stringToBigint(*rawAccount.Balance)
+		if err != nil {
+			return err
+		}
+	}
+	if rawAccount.Kvs != nil {
+		for key, value := range *rawAccount.Kvs {
+			_key, err := hex.DecodeString(key)
+			if err != nil {
+				return err
+			}
+			worldAccount.Storage[string(_key)], err = hex.DecodeString(value)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	// Do we want to update the code if rawAccount.Code == ""
+	if rawAccount.Code != nil && *rawAccount.Code != "" {
```

Comment:
In setAccount, when it is == "", what's the value? nil?

In updateAccount, we should then set the same value (nil maybe).

Same reasoning applies for the other questions.